vignettes/manual/AddNewComponent.Rmd

```diff
@@ -7,15 +7,16 @@ Step one:
 -----------
 
 Add two new files:  
-* headers/core/X.hpp
-* source/core/X.cpp
+* `headers/core/X.hpp`
+* `source/core/X.cpp`
 
 It is best to copy and modify the corresponding files from an already constructed component.
 
 Step two (within X.hpp):  
 -----------
-Under `private` list the variables that are used in the component and their corresponding type. E.g. 
-* `tseries<unitval> x` to create a [time series](TSeries.html).
+Under `private` list the variables that are used in the component and their corresponding type. 
+E.g. 
```

Comment:
Why this addition? The second (tseries) isn't an example of the first, but rather a second step.

Reply:
It looked a little squished on the documentation to have the three variables on one line so I thought maybe to space them out. Not necessary, though!